tests/core/test_prometheus.py

```diff
@@ -268,7 +270,37 @@ def test_submit_gauge_with_labels(self):
         _l2.value = 'my_2nd_label_value'
         self.check._submit(self.check.metrics_mapper[self.ref_gauge.name], self.ref_gauge)
         self.check.gauge.assert_called_with('prometheus.process.vm.bytes', 39211008.0,
-                                            ['my_1st_label:my_1st_label_value', 'my_2nd_label:my_2nd_label_value'])
+                                            ['my_1st_label:my_1st_label_value', 'my_2nd_label:my_2nd_label_value'],
+                                            hostname=None)
+
+    def test_submit_gauge_with_labels_and_hostname_override(self):
+        ''' submitting metrics that contain labels should result in tags on the gauge call '''
```

Comment:
We should use triple double quotes for docstrings [PEP 257](https://www.python.org/dev/peps/pep-0257/)

> For consistency, always use """triple double quotes""" around docstrings.